iothub/service/samples/PnpServiceSamples/TemperatureController/Microsoft.Azure.Devices.TemperatureController/Program.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
+using Newtonsoft.Json;
+using PnpHelpers;
+
+namespace TemperatureController
+{
+    public class Program
+    {
+        // Get connection string and device id inputs
+        private static readonly string s_hubConnectionString = Environment.GetEnvironmentVariable("IOTHUB_CONNECTION_STRING");
+        private static readonly string s_deviceId = Environment.GetEnvironmentVariable("DEVICE_ID");
+
+        // These are values as defined in DTMI used for device client sample with component.
```

Comment:
Add a link to the json in https://github.com/Azure/opendigitaltwins-dtdl/tree/master/DTDL/v2/samples

I am adding that to the device samples as well

Reply:
That's a good point. I will add for both samples.